buildenv/jenkins/variables/defaults.yml

```diff
@@ -232,17 +227,14 @@ s390x_linux:
     11: 'linux-s390x-normal-server-release'
   node_labels:
     build: 'ci.role.build && hw.arch.s390x && sw.os.rhel.7'
+  extra_configure_options:
+    all: '--enable-jitserver'
```

Comment:
This should be unnecessary once https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/540 is approved and rolled out for all versions.

Reply:
Yes, and it can be removed from x and p linux platforms then.